lib/services/HDInsightJob2/README.md

```diff
@@ -0,0 +1,17 @@
+# Microsoft Azure SDK for Node.js - HDInsight
+
+This project provides a Node.js package for accessing the Azure HDInsight. Right now it supports:
+- **Node.js version: 0.6.15 or higher**
+
+## Features
+
+
+## How to Install
+
+```bash
+npm install azure-hdinsight-jobs
```

Comment:
And is this targeting arm or asm?
If arm then can we name this as azure-arm-hdinsight-jobs ?